src/optimizer/OptRule.h

```diff
@@ -50,6 +50,10 @@ class OptRule {
         bool eraseCurr{false};
         bool eraseAll{false};
         std::vector<OptGroupExpr *> newGroupExprs;
+        static const TransformResult &noTransform() {
+            static TransformResult kNoTrans{false, false, {}};
+            return kNoTrans;
+        }
```

Comment:
```
static method -> method -> static variable -> variable
```
Emm, the static method should be place at the beginning of the class/struct.

Reply:
Thanks